data/schema.yaml

```diff
@@ -122,17 +117,17 @@ components:
                   enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
```

Comment:
`Headline`, `Text`, `Image` should be `headline`, `text`, `image`.

Reply:
Thank you, this change has been made now.